src/framework/midi/midievent.h

```diff
@@ -95,17 +89,22 @@ struct Event {
         setOpcode(opcode);
     }
 
-    ME_DEPRECATED Event(channel_t ch, EventType type)
+    Event(Ms::MidiCoreEvent event)
```

Comment:
This constructor and the `toMSEvent` method are equivalent to methods that have been marked deprecated and intended to get rid of.